src/runtime_src/core/common/drv/kds_core.c

```diff
@@ -75,7 +75,7 @@ ssize_t show_kds_custat_raw(struct kds_sched *kds, char *buf)
 ssize_t show_kds_scustat_raw(struct kds_sched *kds, char *buf)
```

Comment:
In all reality, this function's definition should pass in the size of the buffer instead of "assuming" that it is a PAGE_SIZE.

Reply:
Actually kernel default sysfs buffer size is one page, or PAGE_SIZE bytes. Hence, we need to assume or hard code the size in caller or inside this function.